app/src/main/res/values/strings.xml

```diff
@@ -158,16 +158,16 @@
     <string name="app_loyalty_card_keychain">Loyalty Card Keychain</string>
 
     <string name="privacy_policy">Privacy Policy</string>
-    <string name="privacy_policy_popup_text">Privacy policy notice (required by some app stores):\n\nWe collect NO DATA AT ALL, which anyone can confirm since our app is libre software.</string>
+    <string name="privacy_policy_popup_text">Privacy policy notice (required by some app stores):\n\nNO DATA IS COLLECTED AT ALL, which anyone can confirm since our app is libre software.</string>
```

Comment:
Remove?

Reply:
Nah, I'm keeping the string translatable, just in case Google ever decides something stupid like this too.

Reply:
Considering they would have to comply themselves, I think rather an unlikely turn of events.
Anyway, seems good now?

Reply:
Your translation is fine yeah.